app/dashboard/views.py

```diff
@@ -307,7 +307,12 @@ def new_interest(request, bounty_id):
     approval_required = bounty.permission_type == 'approval'
     if approval_required:
         msg = _("You have applied to start work.  If approved, you will be notified via email.")
-
+    elif not approval_required and bounty.bounty_reserved_for_user != profile:
```

Comment:
Hmm this could work ! You let others apply for work letting funders know there are others wanting to work on it  + encourages the reserved user to make a decision quicker on whether they'd be working on it